src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -878,13 +879,11 @@ public static bool TryGetEntityMetadata(CallNode callNode, TexlBinding binding,
 
         // Fetch the description associated with the specified parameter name (which must be the INVARIANT name)
         // If the param has no description, this will return false.
-        public virtual bool TryGetParamDescription(string paramName, out string paramDescription)
+        public virtual bool TryGetParamDescription(string paramName, out string paramDescription, string locale = null)

```

Comment:
>string [](http://example.com/codeflow?start=98&length=6)

How do we decide between whether we pass a CultureInfo or a string?

Reply:
`StringResources.TryGet` expects a string so I'm keeping simple and consistent.